cohorts/views.py

```diff
@@ -2369,14 +2373,13 @@ def cohort_files(request, cohort_id, limit=20, page=1, offset=0, build='HG38', a
             stop = time.time()
             logger.info("[STATUS] Time to get file-list: {}s".format(str(stop - start)))
 
-            if not files_only:
+            counts = {};
+            if do_filter_count:
                 start = time.time()
-                filter_counts = count_public_data_types(
-                    request.user, count_base_clause, inc_filters, cohort_programs,
-                    (type is not None and type != 'all'), build
-                )
+                counts = count_public_data_type(request.user, count_base_clause,
+                                            inc_filters, cohort_programs, (type is not None and type != 'all'), build)
                 stop = time.time()
-                logger.info("[STATUS] Time to count public data files: {}s".format(str(stop-start)))
+                logger.info("[STATUS] Time to count public data files: {}s".format(str((stop-start)/1000)))
```

Comment:
time.time() gives values in seconds, so this should be removed (datetime gives milliseconds though)